styles/globals.scss

```diff
@@ -1,9 +1,14 @@
 @import "./abstracts/variables";
-@import "./components/navigation";
 @import "./base/base";
 @import "./base/typography";
+@import "./components/navigation";
+@import "./components/stamp";
+@import "./components/socialMedia";
+@import "./components/sitemap";
 @import "./components/title";
 @import "./components/subtitle";
+@import "./layout/container";
+@import "./layout/footer";
```

Comment:
To import the components is inefficient to import one by one, better create an expression for importing all files inside components instead.

Reply:
Could you give me more details please